index.js

```diff
@@ -0,0 +1,47 @@
+const inquirer = require('inquirer');
```

Comment:
Should this file be in `/demofile_parsing/` I can't get any node stuff to work outside that folder.

Reply:
From my end the `index.js` file works outside of demofile-parsing. We can move it if you guys think that's best practice though but it should be working outside _demofile-parsing_